tasks/system/Linux.yml

```diff
@@ -1,27 +1,66 @@
 ---
-- name: "Install contrib hybris components"
+- name: 'Install pip package'
```

Comment:
These pre-requisites should be moved to pre_tasks or to prepare stage. It's not scope of hybris-config role to install boto.